src/components/CartItem/index.js

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+
+import useStore from '../../hooks/useStore';
+import StyledButton from '../Button/styled';
+import StyledProductName from '../ProductName/styled';
+import StyledProductPrice from '../ProductPrice/styled';
+
+import StyledCartItem from './styled';
+
+export default function CartItem({productId, imgUrl, name, price}) {
+	const deleteFromCart = useStore(state => state.deleteFromCart);
+
+	return (
+		<StyledCartItem>
+			<Image alt={''} src={imgUrl} width={100} height={100} layout="fixed" />
```

Comment:
The image should have a valid alt

Reply:
Hey Petros, Thank you for your comment. The "alt" texts are empty on advice from Nele (see her comment above). The database is still being expanded, then the "alt" text comes in. Otherwise every product would have the same text. 😄

The merge conflicts are solved very quickly, are only small things. 😉

Reply:
@LudiFlow  Top. grune haken ist dran 👍🏼